compiler/src/iree/compiler/DispatchCreation/BubbleUpExpandShapes.cpp

```diff
@@ -136,9 +137,10 @@ void BubbleUpExpandShapesPass::runOnOperation() {
           return false;
         }
 
-        // If producer generic op is elementwise op, bubble up the expand shape
-        // past this operation.
         if (auto producerGenericOp = dyn_cast<linalg::GenericOp>(producer)) {
+          if (enableBubbleUpExpandShapesAcrossReductionOps)
+            return true;
```

Comment:
nit: we can add trivial braces to be consistent with the coding style of the rest of the file e.g. L129 and L135,"